docs/conf.py

```diff
@@ -225,6 +225,12 @@
         'abort_on_example_error': True
     }
 
+    # Filter out backend-related warnings as described in
+    # https://github.com/sphinx-gallery/sphinx-gallery/pull/564
+    warnings.filterwarnings("ignore", category=UserWarning,
+                            message='Matplotlib is currently using agg, which is a'
```

Comment:
I am just wondering if we should relax the regex to match all the backends it might complain about, not just `agg`.

Reply:
This is the recommended warning catching in the sphinx-gallery docs, so I just thought I'd go with that.